.travis.yml

```diff
@@ -135,46 +137,51 @@ before_install:
     sudo apt-get update -qq
     sudo apt-get install -qq libsdl1.2-dev libsdl-gfx1.2-dev libsdl-mixer1.2-dev libsdl-ttf2.0-dev
   elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-    brew update
-    brew install sdl sdl_gfx sdl_mixer sdl_ttf
+    # SDL installation is handled by Homebrew addon above
     # work around https://stackoverflow.com/questions/51034399/ for now
     (cd /usr/local/include && ln -sf SDL/SDL.h)
   else
-    exit 'Unsupported OS'
+    echo 'Unsupported OS'
+    exit 1
   fi
 - |-
   # Install GLEW if asked for
   if [[ "$HYPERROGUE_USE_GLEW" == "1" ]]; then
     if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       sudo apt-get install -qq libglew-dev
-    elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-      brew install glew
-    else
-      exit 'Unsupported OS'
+    elif [[ "$TRAVIS_OS_NAME" != "osx" ]]; then
```

Comment:
I'd prefer to keep these if-elif ladders with all `==` conditions, so that it's easy to see which OSes are handled and which aren't. In other words, we have a case here for `linux` and a case for `osx`, and then everything else goes to the `else` case. If there's nothing to do for `osx`, then I'd just write

    elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
        # nothing to do
    else
        exit 'Unsupported OS'
    fi

(EDIT: oops, I wrote `!= "osx"` instead of `== "osx"` the first time! Evidence that keeping all the comparisons the same way around is a good idea.)